### PR TITLE
Add caddy for Windows 10

### PR DESCRIPTION
**Pull request by NorbertLuszkiewicz**

### CODE REVIEW

File: install-caddy.ps1 [hunk 1]
```diff
@@ -0,0 +1,112 @@
+﻿#!/usr/bin/env pwsh
```
Comment (solderjs) [inline install-caddy.ps1:1]: These two test files (`install-caddy.{sh,ps1}`) and the `install.bat` should be removed from the commit.

You can either make another commit either
- do a `git rebase -i 2` to squash them away
- or you can do a `git reset HEAD~1` and re-commit without them

(either way you'll need to `git push --force` afterwards to overwrite the commits that have them)

File: caddy/install.ps1 [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    # Unpack archive file into this temporary directory
+    # Windows BSD-tar handles zip. Imagine that.
+    echo "Unpacking $pkg_download"
+    & tar xf "$pkg_download"
+    & dir
+
+    # Settle unpacked archive into place
+    echo "Install Location: $pkg_src_cmd"
+    New-Item "$pkg_src_bin" -ItemType Directory
```
Comment (solderjs) [inline caddy/install.ps1:47]: Did you test this on Windows? I thought that the Caddy releases didn't have subdirectories? Am I mistaken?

To test you should be able to run this in cmd.exe:

```cmd
node _webi/test.js caddy/
powershell -ExecutionPolicy Bypass install-caddy.ps1
```

**Comment by coolaj86** [comment]
@NorbertLuszkiewicz Any update?

File: caddy/install.ps1 (hunk 2 above)
Comment (coolaj86) [reply caddy/install.ps1:47]: Did you perhaps delete it? You may need to run `node _webi/test.js caddy/` again.

**Comment by coolaj86** [comment]
> I removed the files and push only install.p1,
> > The term 'install-caddy.ps1' is not recognized as the name of a cmdlet, function, **script file**

Sounds like you deleted it.

**Comment by coolaj86** [comment]
@NorbertLuszkiewicz are you still working on this?

**Comment by coolaj86** [comment]
This has become time sensitive so I'm closing this out in favor of https://github.com/webinstall/packages/pull/106, as it was not tested before PR and did not work when tested.
